app/views/landmarks/index.html.erb

```diff
@@ -1,7 +1,10 @@
+<h1>Search</h1>
 <%= form_tag landmarks_path, method: "get" do %>
   <%= text_field_tag :search, params[:search], placeholder: "Search Landmarks" %>
   <%= submit_tag "Search", name: nil %>
 <% end %>
+<br><br>
```

Comment:
Don't use `<br>` to create space, that's what CSS is for. Get some divs up in this piece and using the Foundation grid system.